src/components/shared/InputText.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div :class="{ 'flex flex-col gap-2': label }">
+    <label v-if="label" class="text-xs">{{ label }}</label>
+    <input
+      v-if="type === 'text'"
+      :type="type"
+      class="input"
+      :class="{
+        'border-red-400': error,
+        'border-blue-40 focus:border-purple-400': !error
+      }"
+      :placeholder="placeholder"
+      :value="customValue ? customValue : ''"
+    />
```

Comment:
Because I have this input component in other forms as well (I mean, besides the auth forms) and in those forms they're of type text and as for now - they're not supposed to be validated. I may add some validation so they can't be empty in the future but even if so - I will have to add a separate `Yup` thing to be able to do that. Also, the "Can't be empty" error will be displayed in those as a placeholder/value since that's what my mockup implies.